### ISSUE TITLE
unable to parse git version

### ISSUE THREAD

**Issue by furlongm** [NeCTAR-RC/hivemind_contrib#1]
```
Traceback (most recent call last):
  File "/usr/local/bin/hivemind", line 9, in <module>
    load_entry_point('hivemind==0.1dev', 'console_scripts', 'hivemind')()
  File "/home/ubuntu/hivemind/hivemind/common.py", line 592, in main_plus
    shell.shell(parser)
  File "/home/ubuntu/hivemind/hivemind/shell.py", line 85, in shell
    sh.cmdloop()
  File "/home/ubuntu/hivemind/hivemind/shell.py", line 46, in cmdloop
    self.hivemind.execute()
  File "/home/ubuntu/hivemind/hivemind/shell.py", line 73, in execute
    return common.execute_args(self.state, cmd)
  File "/home/ubuntu/hivemind/hivemind/common.py", line 570, in execute_args
    **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/fabric/tasks.py", line 427, in execute
    results['<local-only>'] = task.run(*args, **new_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/fabric/tasks.py", line 174, in run
    return self.wrapped(*args, **kwargs)
  File "/home/ubuntu/hivemind/hivemind/decorators.py", line 38, in wrapper
    return func(*args, **kwargs)
  File "/home/ubuntu/hivemind_contrib/hivemind_contrib/packaging.py", line 176, in buildpackage
    version = git_version()
  File "/home/ubuntu/hivemind_contrib/hivemind_contrib/packaging.py", line 64, in git_version
    raise Exception("Unable to parse version %s" % version)
Exception: Unable to parse version 1.0.0.0b2-167-g622a612
```

**Comment by russell** [comment]
Hivemind buildpackage only support the semantic version style of version numbers.  What version numbering scheme are you trying to use?

**Comment by furlongm** [comment]
1.0.0.0b2-167-g622a612

**Comment by russell** [comment]
The error message is obvious in what is wrong.  The version number doesn't conform to the scheme that it is aware of how to parse.  You haven't asked a question, all you have done is post some information?

You could create a new tag that does conform to a semantic version scheme by using git tag it should allow you to work past this error.
e.g.

```
git tag 1.0.0
```

**Comment by furlongm** [comment]
Ah, I thought the question was implied by the issue title :)

Can `GIT_DESCRIBE_VERSION_REGEX` be fixed to support versions of the form `1.0.0.0b2-167-g622a612` ?

**Comment by sorrison** [comment]
fixed along time ago....
